packages/devtools_app/lib/src/timeline/flutter/event_details.dart

```diff
@@ -41,53 +31,36 @@ class _EventDetailsState extends State<EventDetails> {
         ),
         const PaddedDivider.thin(),
         Expanded(
-          child: Container(
-            child: Stack(
-              children: [
-                if (selectedEvent.isUiEvent) CpuProfilerView(),
-                if (!selectedEvent.isUiEvent) EventSummary(selectedEvent),
-              ],
-            ),
-          ),
+          child: selectedEvent.isUiEvent
+              ? CpuProfilerView()
+              : EventSummary(selectedEvent),
         ),
       ],
     );
   }
 }
 
-class EventSummary extends StatefulWidget {
-  const EventSummary(this.event);
-
-  final TimelineEvent event;
+class EventSummary extends StatelessWidget {
+  EventSummary(this.event)
+      : _connectedEvents = event.isAsyncEvent
+            ? [
```

Comment:
nit: you can put an 
`if` inside the list to take advantage of list comprehensions.

Reply:
done.